src/actions/index.ts

```diff
@@ -328,3 +329,33 @@ export const sendInvite: AsyncActionCreator = (inviteEmail: string) => {
     inviteEmail
   );
 };
+
+export const mint: AsyncActionCreator = (uid: Uid, amount: string) => {
```

Comment:
We can put this in a "money.ts" file or something just to keep `actions.ts` from expanding even more?